spec/models/schedule_spec.rb

```diff
@@ -27,5 +27,17 @@
       schedule = Schedule.new(user_id: 1, start_date: @today, name: "test")
       expect(schedule.valid?).to be false
     end
+
+    it "creates seven workdays if date range is Nov 6 to Nov 12" do
+      schedule = Schedule.new(user_id: 1, start_date: "6/11/2015".to_datetime,
+        end_date: "12/11/2015".to_datetime, name: "test workdays")
+      expect{ schedule.save }.to change{Workday.count}.by 7
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.